env.bzl

```diff
@@ -13,6 +13,18 @@ py_runtime(
 )
 """
 
+def _label_from_condabin(rctx, exe_name):
+    return Label("@{}//:{}/condabin/{}{}".format(rctx.attr.conda_repo,
+        rctx.attr.conda_dir,
+        exe_name,
+        CONDA_EXT_MAP[get_os(rctx)]))
```

Comment:
Just to be extra sure: is `mamba` executable on Windows called `mamba.bat`? That's the case for `conda` for sure (it's called `conda.bat`). 

Also, it's weird that this label is even resolvable with `mamba` because only `conda` executable is exported in `conda.bzl`.

Reply:
Oh, I see now - you are not resolving it with `rctx.path` later. Then I think it shouldn't work as you can't execute a `Label`.

Reply:
> Just to be extra sure: is mamba executable on Windows called mamba.bat? That's the case for conda for sure (it's called conda.bat).

Yes, it is [here](https://github.com/mamba-org/mamba/blob/bff16c2bdc4103ba74c23ab4fdbf58849a55981c/mamba/mamba/shell_templates/win_redirect/mamba.bat) in their source tree. And the Windows GitHub Action is succeeding (but I'm glad we are double checking)

> Oh, I see now - you are not resolving it with rctx.path later. Then I think it shouldn't work as you can't execute a Label.

I do resolve it with `rctx.path` later at this spot: https://github.com/spietras/rules_conda/pull/22/files#diff-b099394c9b99840ac81c083a1d1b3e19dae06815f423e35581374ce69feaf1f6R51